### DISCUSSION TITLE
Does apollo client support transactions?

### DISCUSSION THREAD

**Discussion by ClaudiaGiv** [Akryum/vue-apollo#1088] [Q&A]
I have a nuxtjs app that uses apollo module for graphql communication with FaunaDB.
I want to make multiple mutations in one transaction to FaunaDB. So I don't want to send a different request for every single mutation to FaunaDB, I want to make a single request that contains all the mutations. Is it possible?

**Comment by Akryum** [comment]
Put multiple mutations in one `mutation` document, just like you would ask for multiple fields or multiple root queries.

**Comment by dmitry** [comment]
If you want to have true transaction with rollbacks then you have to create special mutation on the GraphQL backend server.
